.env.example

```diff
@@ -60,6 +60,16 @@ DB_LIST_FOREIGN_KEYS=true
 # LYCHEE_DIST_URL="dist/"
 # LYCHEE_SYM_URL="sym/"
 
+# Authentication guard. This defines how the users are actually retrieved
+# out of your database or other storage mechanisms used by this application
+# to persist your user's data.
+# Possible values are:
+#
+# - session: user is retrieved from session
+# - token: user is retrieved from token like basic auth
```

Comment:
It is already recommended in [auth.php](https://github.com/LycheeOrg/Lychee/blob/master/config/auth.php#L33) but it seems to be a default text from Laravel as i can see the same text in other projects as well. I don't mind the options and possibilities as long as it does allow to disable tokens.

If you have it a boolean it would require some logic in auth.php to be added. My PR currently just allows any of the existing settings to be set.